dist/aem.js

```diff
@@ -519,6 +519,26 @@ async function fetchPlaceholders(prefix = 'default') {
   return window.placeholders[`${prefix}`];
 }
 
+/**
+ * Wrap inline text content of block cells within a <p> tag.
+ * @param {Element} block the block element
+ */
+// eslint-disable-next-line import/prefer-default-export
+function wrapTextNodes(block) {
+  block.querySelectorAll(':scope > div > div').forEach((blockColumn) => {
+    if (blockColumn.hasChildNodes()) {
+      const hasWrapper = !!blockColumn.querySelector('picture')
+        || /* exclude certain elements from being wrapped */ (!!blockColumn.firstElementChild
+          && window.getComputedStyle(blockColumn.firstElementChild).display === 'block');
```

Comment:
we should not use `window.getComputedStyle()` as that would allow stylesheets to interfere with the logic, e.g. setting an `<a>` to `display: block` would cause it not to be wrapped in a `<p>`.